release-content/0.15/migration-guides/10193_Cosmic_text.md

```diff
@@ -0,0 +1,5 @@
+- `Text2dBounds` has been replaced with `TextBounds`, and it now accepts `Option`s to the bounds, instead of using `f32::INFINITY` to inidicate lack of bounds
```

Comment:
```suggestion
- `Text2dBounds` has been replaced with `TextBounds`, and it now accepts `Option`s to the bounds, instead of using `f32::INFINITY` to indicate lack of bounds
```